tasks/www.rb

```diff
@@ -44,6 +44,7 @@
   task :assemble do
     Log.section 'Copy only tutorial into middleman build directory'
     sh('rsync -a --exclude=index.html www/tutorial/dist/* www/build/')
+    sh('cp www/tutorial/dist/index.html www/build/tutorial.html')
```

Comment:
Do we need the other files as well? https://github.com/chef/inspec/blob/master/tasks/www.rb#L26-L29